pfl/algorithm/base.py

```diff
@@ -93,7 +96,7 @@ def load(self, dir_path: str) -> None:
             state = json.load(f)
             # Resume the next iteration.
             self._current_central_iteration = state[
-                'current_central_iteration'] + 1
+                'current_central_iteration']
```

Comment:
I discovered this when making save&restore test for algorithm. algorithm only returns in beginning of next central iteration (when contexts are None), so the current iteration is not actually run before return, so we should start at that iteration if resumed